packages/Spinner/src/Spinner.js

```diff
@@ -1,27 +1,18 @@
 import React from "react";
 import PropTypes from "prop-types";
-import classNames from "classnames";
```

Comment:
This can be removed then https://github.com/acl-services/paprika/blob/03ddf125b7f2e408eaae14822d4e7aeec5cc32bb/packages/Spinner/package.json#L23